### ISSUE TITLE
Feature guard accessibility

### ISSUE THREAD

**Issue by jschwe** [servo/servo#47185]
Feature guard accessibility - This should also make accesskit conditional. Note, that the feature should be defaulted via the `default_web_features` feature (I'm not 100% sure if the feature group name is fitting, but I guess up for discussion)
